### ISSUE TITLE
Masks not working on `<g>` elements.

### ISSUE THREAD

**Issue by atrope** [dnfield/flutter_svg#241]
Hello, can you please take a look in this SVG and help realize why it is not loading?
It does not fire any error.
here is the SVG:
```
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24" viewBox="0 0 24 24">
    <defs>
        <path id="a" d="M17.438 8.438c.31 0 .562.252.562.562v7.313A1.69 1.69 0 0 1 16.312 18H1.688A1.69 1.69 0 0 1 0 16.312V9a.563.563 0 0 1 1.125 0v7.313c0 .309.252.562.563.562h14.624c.31 0 .563-.253.563-.563V9c0-.31.252-.563.563-.563zM9 0c.169 0 .316.079.418.196l.005-.004 3.938 4.5c.082.103.139.229.139.37 0 .311-.252.563-.563.563a.552.552 0 0 1-.418-.196l-.005.004L9.563 2.06v9.752a.563.563 0 0 1-1.126 0V2.06L5.486 5.433a.582.582 0 0 1-.423.192.563.563 0 0 1-.563-.563c0-.141.057-.267.144-.366l-.005-.004 3.938-4.5A.582.582 0 0 1 9 0z"/>
    </defs>
    <g fill="none" fill-rule="evenodd" transform="translate(3 3)">
        <mask id="b" fill="#fff">
            <use xlink:href="#a"/>
        </mask>
        <use fill="#727272" transform="rotate(90 9 9)" xlink:href="#a"/>
        <g fill="#8E93A1" mask="url(#b)">
            <path d="M-3-3h24v24H-3z"/>
        </g>
    </g>
</svg>
```

I Tried usvg generating:

```
<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" xmlns:usvg="https://github.com/RazrFalcon/usvg" usvg:version="0.8.0">
    <defs>
        <mask id="b" x="-0.1" y="-0.1" width="1.2" height="1.2">
            <path fill="#ffffff" fill-rule="evenodd" d="M 17.438 8.438 C 17.748 8.438 18 8.69 18 9 L 18 16.313 C 17.99834725871 17.24440923535 17.24341005121 17.99889920517 16.312 18 L 1.688 18 C 0.75620021668 17.99889792932 0.00110207068 17.24379978332 0 16.312 L 0 9 C 0.01271270943 8.69855860173 0.26079065383 8.46072235233 0.5625 8.46072235233 C 0.86420934617 8.46072235233 1.11228729057 8.69855860173 1.125 9 L 1.125 16.313 C 1.125 16.622 1.377 16.875 1.688 16.875 L 16.312 16.875 C 16.622 16.875 16.875 16.622 16.875 16.312 L 16.875 9 C 16.875 8.69 17.127 8.437 17.438 8.437 Z M 9 0 C 9.169 0 9.316 0.079 9.418 0.196 L 9.423 0.192 L 13.361 4.692 C 13.443 4.795 13.5 4.921 13.5 5.062 C 13.5 5.373 13.248 5.625 12.937 5.625 C 12.77572417052 5.6238681172 12.62300981305 5.55226042805 12.519 5.429 L 12.514 5.433 L 9.563 2.06 L 9.563 11.812 C 9.56299999183 12.12293630838 9.31093630838 12.3749999852 9 12.3749999852 C 8.68906369162 12.3749999852 8.43700000817 12.12293630838 8.437 11.812 L 8.437 2.06 L 5.486 5.433 C 5.37775998399 5.5529360201 5.22453705399 5.62248401669 5.063 5.625 C 4.75206368585 5.625 4.5 5.37293631415 4.5 5.062 C 4.5 4.921 4.557 4.795 4.644 4.696 L 4.639 4.692 L 8.577 0.192 C 8.68524001601 0.0720639799 8.83846294601 0.00251598331 9 0 Z"/>
        </mask>
    </defs>
    <path fill="#727272" fill-rule="evenodd" transform="matrix(0.00000000000000006123233995736766 1 -1 0.00000000000000006123233995736766 21 3)" d="M 17.438 8.438 C 17.748 8.438 18 8.69 18 9 L 18 16.313 C 17.99834725871 17.24440923535 17.24341005121 17.99889920517 16.312 18 L 1.688 18 C 0.75620021668 17.99889792932 0.00110207068 17.24379978332 0 16.312 L 0 9 C 0.01271270943 8.69855860173 0.26079065383 8.46072235233 0.5625 8.46072235233 C 0.86420934617 8.46072235233 1.11228729057 8.69855860173 1.125 9 L 1.125 16.313 C 1.125 16.622 1.377 16.875 1.688 16.875 L 16.312 16.875 C 16.622 16.875 16.875 16.622 16.875 16.312 L 16.875 9 C 16.875 8.69 17.127 8.437 17.438 8.437 Z M 9 0 C 9.169 0 9.316 0.079 9.418 0.196 L 9.423 0.192 L 13.361 4.692 C 13.443 4.795 13.5 4.921 13.5 5.062 C 13.5 5.373 13.248 5.625 12.937 5.625 C 12.77572417052 5.6238681172 12.62300981305 5.55226042805 12.519 5.429 L 12.514 5.433 L 9.563 2.06 L 9.563 11.812 C 9.56299999183 12.12293630838 9.31093630838 12.3749999852 9 12.3749999852 C 8.68906369162 12.3749999852 8.43700000817 12.12293630838 8.437 11.812 L 8.437 2.06 L 5.486 5.433 C 5.37775998399 5.5529360201 5.22453705399 5.62248401669 5.063 5.625 C 4.75206368585 5.625 4.5 5.37293631415 4.5 5.062 C 4.5 4.921 4.557 4.795 4.644 4.696 L 4.639 4.692 L 8.577 0.192 C 8.68524001601 0.0720639799 8.83846294601 0.00251598331 9 0 Z"/>
    <g mask="url(#b)" transform="matrix(1 0 0 1 3 3)">
        <path fill="#8e93a1" fill-rule="evenodd" d="M -3 -3 L 21 -3 L 21 21 L -3 21 Z"/>
    </g>
</svg>
```

But it just renders a gray square:
https://prnt.sc/pshj9d

Thanks

**Comment by dnfield** [comment]
Masks aren't being applied to groups properly.

**Comment by dnfield** [comment]
This also suffers from another bug related to transforms on use elements  -but the `usvg` version doesn't have that.

**Comment by dnfield** [comment]
Fixed in v0.14.4

The first example still won't work because of https://github.com/dnfield/flutter_svg/issues/164
